addons/l10n_nl/migrations/3.1/post-migrate_update_taxes.py

```diff
@@ -1,6 +1,19 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 from odoo.addons.account.models.chart_template import update_taxes_from_templates
+from odoo.upgrade import util
```

Comment:
I'm afraid the upgrade utils cannot be used for local scripts. They are not a dependency of Odoo. You will have to use direct SQL.